tests/test_lazyflow/test_operators/test_opBaseFilter.py

```diff
@@ -0,0 +1,79 @@
+import numpy
+import pytest
+import vigra
+
+from lazyflow.operators.filterOperators import OpBaseFilter
+from lazyflow.roi import sliceToRoi
+from lazyflow.graph import InputSlot
+from lazyflow.rtype import SubRegion
+
+
+@pytest.fixture
+def dummy_filter_op(graph):
+    """OpBaseFilter that passes through input"""
+
+    def _filter_func(source, **filter_kwargs):
+        return source
+
+    class OpPassthroughFilter(OpBaseFilter):
+        _scale = InputSlot(value=0.5)
+        minimum_scale = 0.3
+
+        supports_window = True
+
+        name = "OpPassthroughFilter"
+        filter_fn = staticmethod(_filter_func)
+
+        def resultingChannels(self):
+            return 1
+
+    op = OpPassthroughFilter(graph=graph)
+    return op
```

Comment:
```suggestion
class OpPassthroughFilter(OpBaseFilter):
    """OpBaseFilter that passes through input"""

    _scale = InputSlot(value=0.5)
    minimum_scale = 0.3
    supports_window = True
    name = "OpPassthroughFilter"

    @staticmethod
    def filter_fn(source, **kwargs):
        return source

    def resultingChannels(self):
        return 1


@pytest.fixture
def dummy_filter_op(graph):
    return OpPassthroughFilter(graph=graph)
```

Maybe even remove this fixture and create `OpPassthroughFilter` directly from the `graph` fixture in test functions.

Reply:
aye, true. I started off with more test functions and was reusing the fixture.